src/identity/selectors/index.ts

```diff
@@ -64,3 +64,13 @@ export const selectQuartzOrgsStatus = (
 ): AppState['identity']['quartzOrganizations']['status'] => {
   return state.identity.quartzOrganizations.status
 }
+
+export const selectOrgCreationAllowances = (
+  state: AppState
+): AppState['identity']['allowances']['orgCreation'] => {
+  return state.identity.allowances.orgCreation
+}
```

Comment:
I might consider having separate selectors for the two properties. I know that sounds duplicative, but it may help with re-renders not to have the selected property be the entire object.

Reply:
agreed. Since `loadingStatus` is listed as a property of the `orgCreation` object, every time it's value is updated from `NotStarted`-> `Loading`-> to `Done`, it will force the component to re-render. I didn't take that into consideration.